en/asgardeo/docs/references/conditional-auth/passkey-progressive-enrollment-based-template.md

```diff
@@ -0,0 +1,2 @@
+{% set product_name = "WSO2 Identity Server" %}
+{% include "../../../../../includes/references/conditional-auth/passkey-progressive-enrollment-based-template.md" %}
```

Comment:
Any reason to move adaptive template doc to references?

Reply:
Compared to the other conditional scripts this has a specific use case as it is only applicable if you want progressive enrollment enabled in your passkey implementation. The guide to adding this script is already covered in the passkey documentation and we are introducing the methods in the script and how the script works here. 

For the sake of completeness we can add it with the rest of the conditional auth guides but I feel like this is more of a concepts doc rather than a guide.

Reply:
This doc was put back to the guides -> authentication -> Add conditional authentication